src/BigFish/PaymentGateway.php

```diff
@@ -829,6 +829,10 @@ private static function sendRequest($method, Request $request)
 			curl_setopt($ch, CURLINFO_HEADER_OUT, true);
 		}
 
+		if ($request->gatewayProxy != '') {
```

Comment:
Please, change the `$request->gatewayProxy` to `self::getConfig()->gatewayProxy`

It's better if this works independently of the methods (Init, Result, Close...).